src/Sarif.UnitTests/TestData/SarifCurrentToVersionOneVisitor/v2/OneRunWithLogicalLocations.sarif

```diff
@@ -22,7 +22,56 @@
           "kind": "namespace"
         }
       },
-      "results": []
+      "results": [
+        {
+          "ruleId": "TEST1001",
+          "message": {
+            "text": "A result fired against 'collections::list'."
+          },
+          "locations": [
+            {
+              "fullyQualifiedLogicalName": "collections::list",
+              "logicalLocationIndex": 1
```

Comment:
>logicalLocationIndex [](start = 15, length = 20)

? This is a 2018-10-10 file, so `run.logicalLocations` is a dictionary, so ` location` object shouldn't have a `logicalLocationIndex` property.